djreact/djangoapps/trips/models.py

```diff
@@ -0,0 +1,87 @@
+import json
+
+from django.db import models
+from django.db.models import permalink
+
+
+class Host(models.Model):
+    name = models.CharField(max_length=30)
+    slug = models.SlugField(max_length=50, null=True, blank=True)
+    description = models.TextField(null=True, blank=True)
+    verified = models.BooleanField(default=False)
+
+
+class Location(models.Model):
+    name = models.CharField(max_length=30)
```

Comment:
Docstring for this could be like:
"Location/part of the world where the trip would be visiting."